automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/Leonardo.scala

```diff
@@ -107,9 +96,8 @@ object Leonardo extends RestClient with LazyLogging {
     }
 
     def clusterPath(googleProject: GoogleProject,
-                    clusterName: ClusterName,
-                    version: ApiVersion = V1): String = {
```

Comment:
I got rid of `ApiVersion`. All clients are using `v2` now -- `v1` has been deprecated for more than a year. An upcoming Leo PR will formally remove `v1` from our code.